web/frontend/src/lib/error.ts

```diff
@@ -4,7 +4,7 @@ import { notify } from '@viamrobotics/prime';
 const nonUserErrors = new Set(['Response closed without headers']);
 
 export const isServiceError = (error: unknown): boolean => {
-  return error instanceof Object && error && 'message' in error && 'code' in error && 'metadata' in error;
+  return error instanceof Object && error && 'message' in error;
```

Comment:
Why the change here?

Reply:
Ah yeah, should have mentioned. This is incorrect, some ServiceError objects don't have these fields and therefore were being silenced when they shouldn't. What really matters is the `message` field